<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomPlanner</class>
 <widget class="QMainWindow" name="RoomPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>662</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1135</width>
    <height>662</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RoomPlanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:15px;
color: white;
background: grey;

}

QPushButton:hover {
background: #ffffff;
color:black
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>161</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
color: white;
}

</string>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

border-radius:15px;
color: white;
background: grey;

}

QPushButton:hover {
background: #ffffff;
color:black
}</string>
     </property>
     <property name="text">
      <string>Add Rectangle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

border-radius:15px;
color: white;
background: grey;

}

QPushButton:hover {
background: #ffffff;
color:black
}</string>
     </property>
     <property name="text">
      <string>Add Ellipse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

border-radius:15px;
color: white;
background: grey;

}

QPushButton:hover {
background: #ffffff;
color:black
}</string>
     </property>
     <property name="text">
      <string>Draw walls</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>31</y>
      <width>921</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-40</y>
      <width>1181</width>
      <height>701</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1181</width>
      <height>701</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1181</width>
      <height>701</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
background: rgba(82,82,84,1)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>graphicsView</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Floor plan</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
